sycl/test/devicelib/cmath_fp64_test.cpp

```diff
@@ -47,6 +47,12 @@ void device_cmath_test(s::queue &deviceQueue) {
       auto quo_access = buffer4.template get_access<sycl_write>(cgh);
       cgh.single_task<class DeviceMathTest>([=]() {
         int i = 0;
+        T nan = NAN;
+        T minus_nan = -NAN;
+        T infinity = INFINITY;
```

Comment:
Do I understand correctly that this test uses C macro instead of `std::numeric_limits<T>` methods intentionally to check the compatibility with "C"?

Reply:
Not really.  The C macro is just shorter to write.